apps/astarte_appengine_api/Dockerfile

```diff
@@ -28,21 +28,26 @@ ADD . .
 RUN mix do compile, release
 
 # Note: it is important to keep Debian versions in sync, or incompatibilities between libcrypto will happen
-FROM debian:bullseye-slim
+FROM debian:bookworm
 
 WORKDIR /app
 
+RUN chown -R nobody /app
+
 RUN apt-get -qq update
 
 # Set the locale
 ENV LANG C.UTF-8
 
 # We need SSL
-RUN apt-get -qq install libssl1.1
+RUN apt-get -qq install libssl3
```

Comment:
Is updating libssl needed?

Reply:
When i build images with libssl1.1 it breaks and i found that libssl1.1 package does not exist in Debian bookworm. It has been replaced by libssl3. And when I update it to libssl3 everything works and I run astarte in 5 minutes without any problems.

Reply:
Ok! I noticed that phoenix uses just `libssl` (without version), see https://github.com/phoenixframework/phoenix/blob/a66e7c33019a3d564c4f7f2e48b77efb63c54ad7/priv/templates/phx.gen.release/Dockerfile.eex#L71

Could you please try with that one, too?

Reply:
I have change it and it work as well. Changed in all the docker files.